trikGui/accelerometerWidget.cpp

```diff
@@ -46,7 +46,10 @@ AccelerometerWidget::AccelerometerWidget(trikControl::VectorSensorInterface &acc
 
 void AccelerometerWidget::renew()
 {
-	const auto & value = mAccelerometer.read();
+	const auto &value = mAccelerometer.read();
+	if (value.isEmpty()) {
```

Comment:
Please, add logging at least, or better -- some label for user with error description. That is not very obvious UI, we will not understand that gyroscope doesn't work at all.

Reply:
Done. Now there is "Error" literal.